spec/index.bs

```diff
@@ -1016,13 +1016,35 @@ Applications must not rely on any other promise settlement ordering.
 
 ## Coordinate Systems ## {#coordinate-systems}
 
-- Y-axis is up in normalized device coordinate (NDC): point(-1.0, -1.0) in NDC is located at the bottom-left corner of NDC.
-    In addition, x and y in NDC should be between -1.0 and 1.0 inclusive, while z in NDC should be between 0.0 and 1.0 inclusive.
-    Vertices out of this range in NDC will not introduce any errors, but they will be clipped.
-- Y-axis is down in [=framebuffer=] coordinate, viewport coordinate and fragment/pixel coordinate:
-    origin(0, 0) is located at the top-left corner in these coordinate systems.
-- Window/present coordinate matches [=framebuffer=] coordinate.
-- UV of origin(0, 0) in texture coordinate represents the first texel (the lowest byte) in texture memory.
+Rendering operations use the following coordinate systems:
+
+* <dfn lt="NDC">Normalized device coordinates</dfn> (or NDC) have three dimensions, where:
+    * -1.0 &leq; x &leq; 1.0
+    * -1.0 &leq; y &leq; 1.0
+    * 0.0 &leq; z &leq; 1.0
+    * The bottom-left corner is at (-1.0, -1.0, z).
+    * Vertices outside these bounds will not introduce errors, but they will
+        be clipped.
```

Comment:
Except in the z direction if unclippedDepth is used.

Could probably just remove this bullet though, since clipping is a universal concept.

Reply:
I've dropped the bullet point.